l10n/src/test/main.test.ts

```diff
@@ -54,8 +54,7 @@ describe('@vscode/l10n', () => {
 
     it('load from file uri', async () => {
         mock({
-            '/mock-bundle.json': `{ "message": "translated message" }`,
-            'C:\\mock-bundle.json': `{ "message": "translated message" }`
+            [process.platform === 'win32' ? 'C:\\mock-bundle.json' : '/mock-bundle.json']: `{ "message": "translated message" }`,
```

Comment:
These tests were previously failing on Windows

Reply:
odd... they weren't failing before https://github.com/microsoft/vscode-l10n/actions/runs/4834483913/jobs/8615775683

Reply:
🤷‍♂️ for me I think mock-fs was resolving /mock-bundle.json to c\mock-bundle.json automatically and was erroring that the file already exists when creating it